azure/ConsiderationsForServiceDesign.md

```diff
@@ -216,7 +217,7 @@ cannot collide with a resource path that contains user-specified resource ids.
 
 ## Long-Running Operations
 
-Long-running operations are an API design pattern that should be used when the processing of
+Long-running operations (LROs) are an API design pattern that should be used when the processing of
```

Comment:
I know we have discussed this prior but it would be nice to include a little more specificity into what a "significant amount of time" mean. Perhaps something like "the amount of time is call pattern specific and will vary from a service to another. However, the recommendation is for most services the P99 should complete within 10 seconds and P90 around 3 seconds." I see that we cover the numbers in the Guidelines.md file in the definition section but perhaps 1 second P99 is too stringent for most of the services?

This is certainly not critical for this iteration but perhaps once folks start asking for some guidance we can include some numbers.

Reply:
Opened #542 to track.